src/Tizen.Nlp/TidlClass/message.cs

```diff
@@ -0,0 +1,306 @@
+/*
+ * Generated by tidlc 1.2.4.
+ *
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using Tizen.Applications;
+using Tizen.Applications.RPCPort;
+namespace RPCPort
```

Comment:
Why is not this namespace in Tizen.Nlp?
Why not Tizen.Nlp.RCPort ?

Reply:
this is class file was generated by TIDL command line automatically , we can not modify it

Reply:
@zg2nets We can't export any namespaces and classes that do not follow the guidelines as public api. Please find other approches. One of option is you can change the accessiblity of these classes to internal, not public.